dash/development/_r_components_generation.py

```diff
@@ -392,7 +392,7 @@ def write_help_file(name, props, description, prefix, rpkg_data):
             item_text=item_text,
             description=description.replace('\n', ' ')
         ))
-    if rpkg_data is not None:
+    if rpkg_data is not None and rpkg_data.get('r_examples') is not None:
```

Comment:
🐄  minor: `'r_examples' in rpkg_data`

Reply:
That does look cleaner! Will edit and test; thanks for the feedback.

Reply:
fixed in a7883da